### ISSUE TITLE
Should the learning rate be increased after increasing the batch size?

### ISSUE THREAD

**Issue by TianpengBu** [ultralytics/yolov5#10514]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hi author: 

I see that in yolov5, the default learning rate is 0.01 for SGD, and the total batchsize. If I increase the total batchsize by the factor of 2, should I set lr0=2*lr0? 

Best regards,

**Comment by glenn-jocher** [comment]
@TianpengBu no, the LR does not need to be modified when batch size is changed. Instead loss is scaled by batch size, so you can train any batch size at the same LR.

**Comment by TianpengBu** [comment]
Hi, 
Thanks for the quick rely! My experiment results also confirm your statement. 

Best regards

**Comment by glenn-jocher** [comment]
You're welcome, @TianpengBu! I'm glad to hear that your experiment results confirm the statement. If you have any more questions, feel free to ask. Best regards!
